test/src/test/java/org/corfudb/runtime/collections/CorfuStoreShimTest.java

```diff
@@ -1169,7 +1169,8 @@ public void checkProtoDeduplication() throws Exception {
                 corfuRuntime.getTableRegistry().getProtobufDescriptorTable().size();
         final Collection<CorfuRecord<ProtobufFileDescriptor, CorfuStoreMetadata.TableMetadata>>
                 recordsAfterChange = corfuRuntime.getTableRegistry().getProtobufDescriptorTable().values();
-        assertThat(numProtoFiles).isEqualTo(numProtoFilesAfterChange);
+        records = corfuRuntime.getTableRegistry().getProtobufDescriptorTable().values(); // refresh descriptor table
+        assertThat(numProtoFiles).isEqualTo(numProtoFilesAfterChange - 1); // we added new protofile logdata
```

Comment:
There is no change in the versions of the new and old schemas.  So the size of protobufdescriptor table should remain the same.